CaughtUp/app/src/main/AndroidManifest.xml

```diff
@@ -18,6 +18,10 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+        <meta-data
+            android:name="io.fabric.ApiKey"
+            android:value="a0y2v6pCXfyvHgf6HUU8KPdaJ" />
     </application>
 
+    <uses-permission android:name="android.permission.INTERNET" />
```

Comment:
I guess this is not enough for API 23. We have to dynamically request this permission when the user first tries to use the app.